templates/history-downloads.php

```diff
@@ -47,24 +47,20 @@
 				<?php do_action( 'edd_download_history_header_end' ); ?>
 			</tr>
 		</thead>
-		<?php foreach ( $orders as $order ) :
-			foreach ( $order->get_items() as $key => $item ) :
-
-				// Skip over Bundles. Products included with a bundle will be displayed individually
+		<?php
+		foreach ( $orders as $order ) :
+			foreach ( $order->get_items_with_bundles() as $key => $item ) :
+				// We have to skip bundle parent.
```

Comment:
I am not _sure_ that the bundle parent is actually returned in the `get_items_with_bundles()` array--it doesn't seem to be for me--but it doesn't hurt to leave this in.